app/cms-client/src/components/InnovationWeek.js

```diff
@@ -11,14 +14,17 @@ const breadcrumbs = [
 
 const LeftAndMain = loadComponent('LeftAndMain');
 
-const InnovationWeek = () => {
-  const [md, setMd] = useState('# Loading... please wait\n\nMark down work!!!');
-
+const InnovationWeek = ({ match, ...props}) => {
+  const { path } = match;
   return (
     <LeftAndMain>
-      <ReactMarkdown>{md}</ReactMarkdown>
+      <Switch>
+        <Route path={`${path}/bar/:paramOne?/:paramTwo?`} component={Bar} />
+        <Route path={`${path}/foo`} component={Foo} />
+        <Route path={path} component={Index} exact />
+      </Switch>
     </LeftAndMain>
   );
 };
 
-export default InnovationWeek;
+export default withRouter(InnovationWeek);
```

Comment:
This file shows the way I would expect people to set up up their routing going forward.

If you don't want to use routing at all ... you can just omit the `withRouter` router bit and you'll have a plain boring component.